File: Temperature.c
#include <stdio.h>

int main()
{
    double Fah ;
    double Cel ;
    printf("Enter Fahrenheit : ") ;
    scanf("%lf" , &Fah) ;
    Cel = 5.0/9* (Fah-32) ;
    printf("%.4f Fahrenheit = %.4f Celsius",Fah,Cel) ;
    return 0 ;
}

File: Temperature.java
import java.util.Scanner;
public class Temperature {
	public static void main(String[] args) {
      System.out.print("Enter Fahrenheit : ");
		Scanner scan = new Scanner(System.in);
		double Fah = scan.nextDouble();
		double Cel = 5.0/9*(Fah-32);
		System.out.format("%.4f Fahrenheit = %.4f Celsius",Fah,Cel);
	}

}